packages/web-components/fast-components/src/card/index.ts

```diff
@@ -15,8 +27,80 @@ import { CardStyles as styles } from "./card.styles";
     name: "fast-card",
     template,
     styles,
+    shadowOptions: {
+        mode: "closed",
+    },
```

Comment:
None of the other elements in this repo have closed shadow roots - what's the purpose for adding it here?

Reply:
I think the fluent components are using this...maybe a copy/paste error?

Reply:
Yup brought over from fluent Ill remove